[DOC] Add CLI section to user documentation. (#12802)
Scille/parsec-cloud: docs/cli/introduction.rst

.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS

.. _doc_cli_introduction:

============
Introduction
============

The CLI purpose is twofold:

- a bare bone alternative to the GUI (eg: to be able to automate periodic file upload)

> - a way to access specific parsec operations that are not available on the GUI (eg: have a bird's eye view of server usage)

Reviewer: Those specific operations are probably everything related to Administration API (`server/parsec/asgi/administration.py`) so maybe it is better to mention that it allows to access to administrative operations?

Author: i'll add that. I was thinking about the shamir that's available through the cli

Reviewer: Shamir is a special case I think, because we do not expect users to actually set up and restore account via the CLI. Even if it is possible, I would say that the feature is not completely done until it is implemented in the GUI.

Author: I agree with you in that a feature in not complete until it's in the GUI, but nonetheless it happens that some features are in the CLI first there is the shamir, but also the forget-local feature so IMHO it can happen and it can be worth mentioning (I was quite happy to (re) discover the forget-local feature and easily get rid of old test devices ^^)

.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS

.. _doc_cli_introduction:

============
Introduction
============

The CLI purpose is twofold:

> - a bare bone alternative to the GUI (eg: to be able to automate periodic file upload)

Reviewer: ```suggestion
- a bare-bones alternative to Parsec Desktop or Web application (e.g. to automate periodic file imports)
```

> - a way to access specific parsec operations that are not available on the GUI (eg: have a bird's eye view of server usage)

Reviewer: ```suggestion
- a way to access specific Parsec operations that are not available on the GUI (e.g. monitoring server usage)
```

> The cli outputs provided in this documentation are examples. The specific output you have may vary depending on the CLI version.

Reviewer: I would remove it from here, since 1/there are no output in this section and 2/ you already mention it in the "First steps" section.

```suggestion
```
